src/frontend-samples/explode-sample/ExplodeTile.ts

```diff
@@ -0,0 +1,629 @@
+    const maxRange = calculateExplodeTransform(bBox.center, centerOfMass, ExplodeApp.explodeAttributes.max).multiplyRange(bBox);
+    const minRange = calculateExplodeTransform(bBox.center, centerOfMass, ExplodeApp.explodeAttributes.min).multiplyRange(bBox);
+    maxRange.extendRange(minRange);
+    return maxRange;
+  }
+
+  /** Tile has no content and should never request any. */
+  public async requestContent(_isCanceled: () => boolean): Promise<TileRequest.Response> {
+    assert(false, "Tile has no content");
+    return undefined;
+  }
+
+  /** Tile has no content and should never have any to read. */
+  public async readContent(_data: TileRequest.ResponseData, _system: RenderSystem, _isCanceled: () => boolean): Promise<TileContent> {
+    throw new Error("Tile has no content");
+  }
+
+  public pruneChildren(olderThan: BeTimePoint): void {
+    const children = this.children as ExplodedGraphicsTile[];
+    assert(undefined !== children);
+
+    const partitionIndex = partitionArray(children, (child) => !child.usageMarker.isExpired(olderThan));
+
+    // Remove expired children.
+    if (partitionIndex < children.length) {
+      const expired = children.splice(partitionIndex);
+      for (const child of expired)
+        child.dispose();
+    }
+
+    // Restore ordering.
+    children.sort((x, y) => y.toleranceLog10 - x.toleranceLog10);
+  }
+}
+
+/** Produce a number iterating from 0 to the max integer. */
+function* makeIdSequence() {
+  let current = 0;
+  while (true) {
+    current %= Number.MAX_SAFE_INTEGER;
+    yield ++current;
+  }
+}
+const requestIdSequence = makeIdSequence();
+
+/** These tiles actually handle loading and transforming the graphics for each tile at a specific tolerance. */
+export class ExplodedGraphicsTile extends Tile {
+  public get data(): ElementData { return this.parent.data; }
+  public get rootTile(): RootTile { return this.parent.parent as RootTile; }
+  public get centerOfMass(): Point3d { return this.parent.centerOfMass; }
+  public get formatVersion(): number { return this.parent.formatVersion; }
+
+  private _prevExplodeFactor: number = -1;
+  private _explodeTransform = Transform.createIdentity();
+
+  constructor(
+    public readonly parent: ElementTile,
+    public readonly toleranceLog10: number,
+  ) {
+    super({
+      parent,
+      isLeaf: true,
+      contentId: `${parent.contentId}_${toleranceLog10}`,
+      range: parent.range,
+      maximumSize: parent.maximumSize,
+    }, parent.tree);
+
+    this.setExplodeTransform(ExplodeApp.explodeAttributes.min);
+  }
+
+  public computeLoadPriority(_viewports: Iterable<Viewport>): number {
+    // We want the element's graphics to be updated as soon as possible
+    return 0;
+  }
+
+  /** Creates an unique id for requesting tiles from the backend. */
+  private makeRequestId(): string {
+    const requestId = requestIdSequence.next();
+    if (requestId.done)
+      return (-1).toString(16);
+    return requestId.value.toString(16);
```

Comment:
Add a prefix or suffix to requestId to make it unique among all requests (including, e.g., those made by DynamicIModelTile) and explain why that is necessary.

Reply:
Updated.  Please fact check as I could not find documentation on the consequences of matching request ids.